mypy/stubgenc.py

```diff
@@ -218,6 +257,24 @@ def add_typing_import(output: list[str]) -> list[str]:
         return output[:]
 
 
+def get_members(obj: object) -> list[tuple[str, Any]]:
+    obj_dict: Mapping[str, Any] = getattr(obj, "__dict__")  # noqa: B009
+    results = []
+    for name in obj_dict:
+        if is_skipped_attribute(name):
+            continue
+        # First try to get the value via getattr.  Some descriptors don't
+        # like calling their __get__ (see bug #1785), so fall back to
+        # looking in the __dict__.
```

Comment:
This comment seems wrong; the current code discards cases where getattr() fails, it doesn't fall back to the value in `__dict__`.

Reply:
Good catch.  This was adapted from `inspect.getmembers` and I left the comment unaltered.  This is fixed.